USAGE.md

```diff
@@ -6187,7 +6187,7 @@ For more information, please see our [User Guide](https://sendgrid.com/docs/API_
 **This endpoint allows you to disassociate a link whitelabel from a subuser.**
 
 Link whitelables can be associated with subusers from the parent account. This functionality allows
-subusers to send mail using their parent's linke whitelabels. To associate a link whitelabel, the parent account
+subusers to send mail using their parent's like whitelabels. To associate a link whitelabel, the parent account
```

Comment:
Should be "link whitelables" 

Can you please make this change?

Reply:
Done!
Thanks, @mbernier!